CONTRIBUTORS.markdown

```diff
@@ -0,0 +1,50 @@
+Below you can find a list of people that contributed code to the IIS system (in order of appearance) before its move to GitHub (i.e., between September 2012 and September 2015) along with their:
```

Comment:
This is already mentioned in the main README file and I wouldn't like to repeat the same information again. Do you think that this will be enough?